src/components/Notification.tsx

```diff
@@ -0,0 +1,60 @@
+import { useCallback, useEffect } from 'react';
+import { Alert, Link, Snackbar, styled } from '@mui/material';
+
+import { NotificationActionTypes } from '~/providers/NotificationProvider/actions';
```

Comment:
it should be imported as 
```suggestion
import { NotificationActionTypes } from '~/providers';
```

Reply:
https://github.com/0xShishigami/feo-web3-react-boilerplate/pull/2/commits/ba0c3959874ad8a573436f5456e5d6253b4026d2